apps/vr-tests-react-components/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@fluentui/vr-tests-react-components",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 8 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26312&commitId=99374d2dc00673b581f9f5b34c28ab29ab67f279&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1037 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.Required.default.chromium.png|69| Changed|
|ChoiceGroup.Required.hover selected.chromium.png|69| Changed|
|ChoiceGroup.Required.hover unselected.chromium.png|69| Changed|
|ChoiceGroup.Required.selected.chromium.png|69| Changed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>